# goal definition
bool coffe_order
---
# result definition
bool order_success